client/blocks/plugins-update-manager/schedule-list.tsx

```diff
@@ -130,7 +132,9 @@ export const ScheduleList = ( props: Props ) => {
 						canCreateSchedules && (
 							<Text as="p">
 								<Icon className="icon-info" icon={ info } size={ 16 } />
-								The current feature implementation only allows to set up two schedules.
+								{ translate(
+									'The current feature implementation allows to set up two schedules.'
```

Comment:
Let's try mentioning that in the empty state message so we don't have to display that. Even in the revised wording it sounds like we're apologizing that this is limited to two.

Reply:
If we end up keeping this in, I'd suggest:

`This feature is currently limited to two schedules.`